src/Symfony/Component/Lock/Bridge/DynamoDb/Tests/Functional/Store/AbstractStoreTestCase.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock\Bridge\DynamoDb\Tests\Functional\Store;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Lock\Exception\LockConflictedException;
+use Symfony\Component\Lock\Key;
+use Symfony\Component\Lock\PersistingStoreInterface;
+
+/**
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+abstract class AbstractStoreTestCase extends TestCase
```

Comment:
Instead of duplicating this base test case, the class should be moved to `Symfony\Component\Lock\Test\StoreTestCase` (`Test` without `s`), so that the class is exposed by the `symfony/lock` package, and becomes part of its API. Then it can be used by all the bridge packages.

Reply:
We don't prefix the shared TestCase with `Abstract` in [other components](https://github.com/search?q=repo%3Asymfony%2Fsymfony+path%3A**%2F*TestCase.php&type=code). You can remove it.